### ISSUE TITLE
Non Damaging Moves

### ISSUE THREAD

**Issue by skhati1** [skhati1/442_BWK#2]
There exist moves to which features have not been implemented - Ex: Disable, Mimic. If this would be functionality in another version, the app can be changed so that when a quick random battle is initiated, Pokemons in the field do not have these moves.

**Comment by brandaogabe** [comment]
I agree no need to keep the moves view able to the user if they are not usable.
